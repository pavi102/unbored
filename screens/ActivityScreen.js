import React from 'react'
import { Text, StyleSheet, View } from 'react-native'


const ActivityScreen = () => {
    const activities = ["Go out for a walk", "Make some tea", "Meditate for 10 minutes", "Do 20 pushups", "Learn a new breakfast recipe"]
    let randomActivity = activities[Math.floor(Math.random() * activities.length)]
    return (
        <View style={styles.container}>
            <View style={styles.textContainer}>
                <Text style={styles.mainText}>{randomActivity}</Text>
            </View>

        </View >
    )
}

const styles = StyleSheet.create({
    container: {
        flex: 1,
        backgroundColor: '#34393b',
        alignItems: 'center',
    },
    textContainer: {
        backgroundColor: '#34393b',
        margin: '10%',
        alignItems: 'center',
    },
    mainText: {
        fontSize: 30,
        fontFamily: 'monospace',
        color: '#a3a6ad',
        alignSelf: 'center',


    },
})

export default ActivityScreen;