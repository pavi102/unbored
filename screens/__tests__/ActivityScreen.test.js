import React from 'react';
import {cleanup, render} from '@testing-library/react-native'
import ActivityScreen from '../ActivityScreen'

describe('Activity Screen', () => {
    afterEach(cleanup)
    it('should render correctly', () => {
        const toJson = render(<ActivityScreen />)
        expect(toJson).toMatchSnapshot();
    })
})