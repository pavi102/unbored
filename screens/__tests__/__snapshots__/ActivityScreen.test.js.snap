// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Activity Screen should render correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#34393b",
      "flex": 1,
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#34393b",
        "margin": "10%",
      }
    }
  >
    <Text
      style={
        Object {
          "alignSelf": "center",
          "color": "#a3a6ad",
          "fontFamily": "monospace",
          "fontSize": 30,
        }
      }
    >
      Do 20 pushups
    </Text>
  </View>
</View>
`;
